src/js/components/Graphs/Legend/legend.spec.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import EnzymeAdapter from 'enzyme-adapter-react-16';
+import { createLegend } from './legendHelper';
+import { legendArraysMock } from './legend.mock';
+import {
+  LegendHeader,
+  LegendSection,
+  LegendItemIcon,
+  SquareIcon,
+  CircleIcon,
+  RectangleIcon,
+  LineIcon,
+  LegendItemDescription,
+  LegendItem,
+} from './legend.style';
+
+Enzyme.configure({ adapter: new EnzymeAdapter() });
```

Comment:
Shouldn't this be created in config file rather than in test?

Reply:
I don't know. Should be? Not every test use enzyme ;p

Reply:
But you don't need to make that in every test, worth it

Reply:
ok, setup file added